### ISSUE TITLE
GNOME 45 support

### ISSUE THREAD

**Issue by D3vil0p3r** [Aylur/gnome-extensions#67]
Hello @Aylur 
Will the extension support GNOME 45 that will be released in the third week of September?

**Comment by Aylur** [comment]
No, I have lost interest in this extension, mostly because I have migrated over to standalone window managers.
If Gnome implements their "mosaic" tiling and I like it, I might come back to Gnome and make an overhaul.

**Comment by D3vil0p3r** [comment]
That's a pity. Your extension got a big fan base over time.

**Comment by D3vil0p3r** [comment]
@Aylur https://github.com/pop-os/shell

**Comment by ghostpandaboo** [comment]
Hi @Aylur , Please don't abandon this extension. Like @D3vil0p3r said, your extension has a huge fan base and you can use Forge extension for tiling in Gnome if you prefer, https://github.com/forge-ext/forge

**Comment by D3vil0p3r** [comment]
It would be interesting to have  a screenshot of how ags would look

**Comment by ispanos** [comment]
@Aylur Please add at the top of this repo that you are not planning to keep it up to date and maybe allow someone to take over. I'm sure there are many people who would be able to take over and at least keep usable for gnome 45. [This](https://youtu.be/q4odN8N24qk?si=3di5wXBnn7SOREis) was the inspiration for this request.

Thank you for creating this great extension. I liked having so many extensions in one place.

**Comment by D3vil0p3r** [comment]
I think his AGS project should be a continuation of this widget, just structured in more modular manner... or not?

**Comment by Aylur** [comment]
> @Aylur Please add at the top of this repo that you are not planning to keep it up to date and maybe allow someone to take over. I'm sure there are many people who would be able to take over and at least keep usable for gnome 45. [This](https://youtu.be/q4odN8N24qk?si=3di5wXBnn7SOREis) was the inspiration for this request.

I am actually planning to make most part of this extension into smaller seperate extensions, but its probably a good idea to update the readme.

**Comment by Aylur** [comment]
> I think his AGS project should be a continuation of this widget, just structured in more modular manner... or not?

That project is incompatible with Gnome, its for wayland compositors that implement the layer-shell protocol, which Gnome sadly doesnt

**Comment by mbeps** [comment]
In that case, we should break this extension into smaller more manageable extensions. Not everyone was using all the features from this extensions, me personally I was using the power menu and quick settings tweaks. This extensions was one of the best

**Comment by DavidKozdra** [comment]
I love this extention its sort of the corner stone to my whole design, any thing I can do ?

**Comment by markocic** [comment]
I tried to port this extension to GNOME 45. https://github.com/markocic/gnome-extensions . As mentioned in my README, Quick Settings Tweaks are disabled since I didn't know how to do it myself. Hope someone finds this helpful or tries to build of of it.

**Comment by bearpawmx** [comment]
Just upgraded to Gnome 45 and lost my desktop clock because I configured it through @Aylur 's extention (along with several other things). Sad day

**Comment by amitkshirsagar13** [comment]
> I tried to port this extension to GNOME 45. https://github.com/markocic/gnome-extensions . As mentioned in my README, Quick Settings Tweaks are disabled since I didn't know how to do it myself. Hope someone finds this helpful or tries to build off of it.

I got your branch and got it working. But had to disable few features.

Battery status media and a few other things do work.

**Comment by Frios-e-lacticinios** [comment]
Please aylur, don't give up on your extension, I love it, currently because I accidentally update my Manjaro Gnome Pro 45 and see that your extension isn't there for it, I'm using Windows 11 (dualboot) because your extension changes Gnome , Gnome is not the same without you, I have never seen an extension as complete and beautiful as this one, gnome is lifeless without it, please don't abandon it :(

(Sorry for my english on google tradutor, i from Brazil :P)

**Comment by markocic** [comment]
> > I tried to port this extension to GNOME 45. https://github.com/markocic/gnome-extensions . As mentioned in my README, Quick Settings Tweaks are disabled since I didn't know how to do it myself. Hope someone finds this helpful or tries to build off of it.
> 
> I got your branch and got it working. But had to disable few features.
> 
> Battery status media and a few other things do work.
> 
> [amitkshirsagar13.tar.gz](https://github.com/Aylur/gnome-extensions/files/13404828/amitkshirsagar13.tar.gz)
> 
> Use below command to unzip and use extension.
> 
> ```
> tar -xvf amitkshirsagar13.tar.gz
> mv widgets@markocic ~/.local/share/gnome-shell/extensions/
> ```

Would you mind sharing more info? I tested my repo with Fedora 39 and I know it's working on Kali Linux from a message I got. 

What distro are you using and what was the error message?

**Comment by amitkshirsagar13** [comment]
> > > I tried to port this extension to GNOME 45. https://github.com/markocic/gnome-extensions . As mentioned in my README, Quick Settings Tweaks are disabled since I didn't know how to do it myself. Hope someone finds this helpful or tries to build off of it.
> > 
> > 
> > I got your branch and got it working. But had to disable few features.
> > Battery status media and a few other things do work.
> > [amitkshirsagar13.tar.gz](https://github.com/Aylur/gnome-extensions/files/13404828/amitkshirsagar13.tar.gz)
> > Use below command to unzip and use extension.
> > ```
> > tar -xvf amitkshirsagar13.tar.gz
> > mv widgets@markocic ~/.local/share/gnome-shell/extensions/
> > ```
> 
> Would you mind sharing more info? I tested my repo with Fedora 39 and I know it's working on Kali Linux from a message I got.
> 
> What distro are you using and what was the error message?

I am using Ubuntu below details.
```
# System Details Report
---

## Report details
- **Date generated:**                              2023-12-23 10:51:22

## Hardware Information:
- **Hardware Model:**                              HP Victus by HP Laptop 16-e0xxx
- **Memory:**                                      32.0 GiB
- **Processor:**                                   AMD Ryzen™ 5 5600H with Radeon™ Graphics × 12
- **Graphics:**                                    NVIDIA GeForce GTX 1650
- **Graphics 1:**                                  NVIDIA GeForce GTX 1650
- **Disk Capacity:**                               1.5 TB

## Software Information:
- **Firmware Version:**                            F.18
- **OS Name:**                                     Ubuntu 23.10
- **OS Build:**                                    (null)
- **OS Type:**                                     64-bit
- **GNOME Version:**                               45.1
- **Windowing System:**                            X11
- **Kernel Version:**                              Linux 6.5.0-14-generic
```

**Comment by iFabse** [comment]
Added GNOME 46 support at https://github.com/iFabse/Aylurs-gnome-extension but only for BatteryBar right now.
